setup.py

```diff
@@ -73,12 +73,12 @@
 
 install_requires = [
     'Flask-BabelEx>=0.9.4',
-    'Flask-Principal>=0.4.0,<0.5.0',
+    'Flask-Principal>=0.4.0',
```

Comment:
should this not be specified by invenio-access now?

Reply:
It comes from invenio-admin. Maybe should be as part of the new deps management?